### PR TITLE
Improve the document

### PR DESCRIPTION
**Pull request by holy12345**
Improve the documentation for the Eureka chapter

### CODE REVIEW

File: docs/src/main/asciidoc/spring-cloud-netflix.adoc [hunk 1]
```diff
@@ -29,7 +29,7 @@ When a client registers with Eureka, it provides meta-data about itself
 such as host and port, health indicator URL, home page etc.  Eureka
 receives heartbeat messages from each instance belonging to a service.
 If the heartbeat fails over a configurable timetable, the instance is
-normally removed from the registry.
+normally removed from the registry(assuming you turn off Eureka's self-protection feature).
```
Comment (fahimfarookme) [inline docs/src/main/asciidoc/spring-cloud-netflix.adoc]: , if self-preservation is turned-off.

**Comment by holy12345** [comment]
@spencergibb Hello, thanks for your reply. I temporarily restore this paragraph. At the same time I @ Configuration, @ ComponentScan, @ EnableAutoConfiguration note replaced by @ SpinBootApplication. What do you think of this

**Comment by holy12345** [comment]
@spencergibb In addition, there are some places in the document using @ Configuration, @ ComponentScan, @ EnableAutoConfiguration, and in some places using @SpringBootApplication. I want to modify it uniformly into @SpringBootApplication. If you agree, I will be unified modification in this pr, thank you

**Comment by holy12345** [comment]
@fahimfarookme Thank you for your reviewed.If I say so, do you think it is OK?
if self-preservation is turned-off or self-preservation is turned-on and renewed not less the percentage configured for the value (not below than 85% in 15 minutes)

**Comment by ryanjbaxter** [comment]
I think you can replace `@Configuration`, `@ComponentScan`, `@EnableAutoConfiguration`with `@springbootapplication`.

**Comment by holy12345** [comment]
@ryanjbaxter Thank you for your reply, I will modify it according to your suggestion.

**Pull request closed without merging**